homeassistant/components/light/mipow.py

```diff
@@ -0,0 +1,208 @@
+"""
+Support for Mipow lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.mipow/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_DEVICES, CONF_NAME
+from homeassistant.components.light import (
+    ATTR_RGB_COLOR, ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS,
+    SUPPORT_RGB_COLOR, Light,
+    FLASH_LONG, ATTR_WHITE_VALUE,
+    FLASH_SHORT,
+    SUPPORT_FLASH, ATTR_FLASH, PLATFORM_SCHEMA, SUPPORT_WHITE_VALUE)
+import homeassistant.helpers.config_validation as cv
+REQUIREMENTS = ['mipow==0.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'mipow'
+
+SUPPORT_MIPOW_LED = (SUPPORT_RGB_COLOR |
+                     SUPPORT_FLASH | SUPPORT_BRIGHTNESS | SUPPORT_WHITE_VALUE)
+
+DEVICE_SCHEMA = vol.Schema({
+    vol.Optional(CONF_NAME): cv.string,
+})
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_DEVICES, default={}): {cv.string: DEVICE_SCHEMA},
+})
+
+# pylint: disable=C0103
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Mipow platform."""
+    lights = []
+    for address, device_config in config[CONF_DEVICES].items():
```

Comment:
OK, I'm fine keeping the address the way it is if that's convention from other platforms.